rmgpy/kinetics/surfaceTest.py

```diff
@@ -137,6 +147,20 @@ def test_pickle(self):
         self.assertAlmostEqual(self.stick.Tmax.value, stick.Tmax.value, 4)
         self.assertEqual(self.stick.Tmax.units, stick.Tmax.units)
         self.assertEqual(self.stick.comment, stick.comment)
+        for key in self.stick.coverage_dependence.keys():
+            match = False
+            for key2 in stick.coverage_dependence.keys():
+                if key.is_identical(key2):
+                    match = True
+            self.assertTrue(match)
+        for species, parameters in self.stick.coverage_dependence.items():
+            match = False
+            for species2 in stick.coverage_dependence.keys():
+                if species.is_identical(species2):
+                    match = True
+                    for key, value in parameters.items():
+                        self.assertEqual(value.value_si, stick.coverage_dependence[species2][key].value_si)
+            self.assertTrue(match)
```

Comment:
do we need to iterate though keys and then iterate through items.  Can we just iterate through items?